plugins/hls-rename-plugin/src/Ide/Plugin/Rename.hs

```diff
@@ -48,10 +48,6 @@ import           Ide.Types
 import           Language.LSP.Server
 import           Language.LSP.Types
 
-instance Hashable Location
-instance Hashable Range
-instance Hashable Position
-instance Hashable UInt
```

Comment:
Instances have been added to lsp-types-1.5. This is not a question or complaint, just a note for the next reviewer